torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2467,9 +2574,27 @@ c10::optional<ScalarType> findDtypeForValue(const torch::jit::Value* v) {
   return c10::nullopt;
 }
 
+static bool isValidIdentifierChar(char c, size_t pos) {
```

Comment:
This will probably conflict with #65010. NBD if your PR lands first, but otherwise you'd have to rebase.

Reply:
Thats completely ok. No hurry. If you have any suggestions for adding tests for my PR, plz let me know.